ChangeLog.md

```diff
@@ -7,6 +7,13 @@ Notable changes to the FreeOrion project will be documented in this file.
 [v0.4.9+] - TBA
 ---------------
 
+### Detailed Changelog
+
+#### Removed features
+
+- Removed the scripting/keymaps.inf configuration file, which allowed users to remap one keycode to another.
+  This functionality is already exposed in a differnt way via configurable keyboard shortcuts.
```

Comment:
differnt -> different

Reply:
Rewritten to describe the original intent of the code.